clean-abap/CleanABAP.md

```diff
@@ -1261,6 +1262,24 @@ especially if you embed multiple variables in a text.
 DATA(message) = `Received an unexpected HTTP ` && status_code && ` with message ` && text.
 ```
 
+### Use && to split long text
+
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Strings](#strings) > [This section](#use--to-split-long-text)
+
+```ABAP
+DATA(message) = |This is a really long text having { variable } and characters | && 
+                |exceeding the preferred limit of { character_limit } with message { text }|.
+```
+
+If the text is long and exceeds the preferred limit of 120 characters, the text should be split 
```

Comment:
This implies the preferred limit of the text is 120 characters, when it should actually refer to the entire line of code.